bot/grenouille_http_server.py

```diff
@@ -0,0 +1,50 @@
+from http.server import BaseHTTPRequestHandler, HTTPServer
+import threading
+import logging
+import cgi
+import os
+
+
+def MakeHandler(bot):
+    class testHTTPServer_RequestHandler(BaseHTTPRequestHandler):
+        
+        def do_POST(self):
+            form = cgi.FieldStorage(
+                fp=self.rfile,
+                headers=self.headers,
+                environ={"REQUEST_METHOD": "POST"}
+            )
+            
+            for item in form.list:
+                if (item.name == "say"):
+                    bot.send_msg(item.value)
+                    
+                    self.send_response(200)
+                    self.send_header("Content-type", "text/html")
+                    self.end_headers()
+                    return
+    return testHTTPServer_RequestHandler
+
+class GrenouilleHttpServer:
```

Comment:
Would be best if you make this inherit from `Thread` and merge all 3 functions. So you would be able to start it similar to other classes. Timer are not the best way to do it.